content/courses/Ada_For_The_C_Embedded_Developer/chapters/06_Translation.rst

```diff
@@ -816,20 +851,138 @@ In C, we would rely on bit-shifting and masking to set that specific bit:
         printf("v = %d\n", v);
     }
 
+.. admonition:: Important
+
+    The :ada:`Default_Value` aspect is associated with type declarations. When
```

Comment:
I would start at a higher level, something like:

Ada has the concept of default initialization, which provides default values for either a type of its components. This is the case for example when specifying default values for components of record:

type X is record
   v : Integer := 0;
end record;

when associating a type with a Default_Value aspect, or even by default on certain types (e.g. access types have a default value of null).